### PR TITLE
clean up main page, new 'about us' and other improves

### PR DESCRIPTION
**Pull request by turbotimon**

### CODE REVIEW

File: sps20/content/aboutus/contents.lr [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Non-profit
+
+We are a non-profit volunteer organization funded entirely by our participants, donators and sponsors. The conference is organized by the Swiss Python Summit Association.
+
+We are always looking for people to [help us organize the conference](/helpus) or [support us with a donation](/helpus#donate).
+
+## Code of Conduct
+
+Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from the conference without a refund at the discretion of the conference organisers.
```
Comment (rnestler) [inline sps20/content/aboutus/contents.lr:19]:
```suggestion
Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from the conference without a refund at the discretion of the conference organizers.

```

File: sps20/content/aboutus/contents.lr [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+See [the full Code of Conduct](/coc).
+
+## History
+
+After the good experience of [EuroPython 2015](https://ep2015.europython.eu/), a small group of enthusiasts came up with the idea to organise a Python conference in Switzerland. A few meetings and discussions later, we organized a small and very inexpensive event in early 2016. We expected about 50 people, but 130 signed up. It was a great success, so we decided to continue organising the conference. Two more editions of the conference were held in 2017 and 2018, and after a short break, another one in 2022. After some new members and a new board were found, the Swiss Python Summit was continued with annual summits from 2023. We look forward to seeing you at the next summit!
```
Comment (rnestler) [inline sps20/content/aboutus/contents.lr:25]:
```suggestion
After the good experience of [EuroPython 2015](https://ep2015.europython.eu/), a small group of enthusiasts came up with the idea to organize a Python conference in Switzerland. A few meetings and discussions later, we organized a small and very inexpensive event in early 2016. We expected about 50 people, but 130 signed up. It was a great success, so we decided to continue organizing the conference. Two more editions of the conference were held in 2017 and 2018, and after a short break, another one in 2022. After some new members and a new board were found, the Swiss Python Summit was continued with annual summits from 2023. We look forward to seeing you at the next summit!
```

File: sps20/content/venue/contents.lr [hunk 3]
```diff
@@ -1,38 +1,51 @@
-title: Venue
+title: Location
```
Comment (rnestler) [inline sps20/content/venue/contents.lr:1]:
```suggestion
title: Venue
```

Whats wrong with venue?

File: sps20/content/venue/contents.lr [hunk 4]
```diff
@@ -1,38 +1,51 @@
...
 
-## Conference Room
-![aula](./aula.jpg)
-
-The conference will take place in the **Aula (building 4)** upstairs. Or just follow the signs on campus.
-## Quiet Room
+The Swiss Python Summit is held on the campus of the OST Campus (formerly HSR) in Rapperswil.
 
-There is a separate room for working or for relaxing. Free wifi as well as power connections are available
+**Address:**
```
Comment (rnestler) [inline sps20/content/venue/contents.lr:15]:
```suggestion
### Address
```

Or do 3rd level headers look strange? Same for the others.

File: sps20/content/venue/contents.lr [hunk 5]
```diff
@@ -1,38 +1,51 @@
...
 ---
-description: A guide to the venue
+description: A guide to the conference venue
 ---
 ordering: 1
 ---
 body:
 
-# Venue
-The Swiss Python Summit is held on the campus of the **[OST Eastern Switzerland University of Applied Sciences](https://www.ost.ch/)** (formerly HSR) in **[Rapperswil](https://www.instagram.com/visitrapperswil/)** at the **Aula** in **Building 4** (upstairs).
```
Comment (rnestler) [inline sps20/content/venue/contents.lr:10]: I kinda liked the `visitrapperswil` link, but I don't mind that much.

**Review by rnestler — APPROVED** [review]
Consistent spelling of organize and a few minor suggestions.

File: sps20/content/aboutus/contents.lr (hunk 2 above)
Comment (turbotimon) [reply sps20/content/aboutus/contents.lr:25]: I thought we agreed on BE. but personally i don't care if BE or AE and as just found out it's a wild mix now..

File: sps20/content/venue/contents.lr (hunk 3 above)
Comment (turbotimon) [reply sps20/content/venue/contents.lr:1]: I lot of other conference side use location and i personally like location more. but i'm fine with both. On going discussion: https://swiss-python-summit.slack.com/archives/CB5962QQJ/p1718131716656189?thread_ts=1718053207.563329&cid=CB5962QQJ

File: sps20/content/venue/contents.lr (hunk 5 above)
Comment (turbotimon) [reply sps20/content/venue/contents.lr:10]: Ok, I will add it back

File: sps20/content/venue/contents.lr (hunk 4 above)
Comment (turbotimon) [reply sps20/content/venue/contents.lr:15]: Done

File: sps20/content/venue/contents.lr (hunk 5 above)
Comment (turbotimon) [reply sps20/content/venue/contents.lr:10]: BTW, how is the "Ask hotels for discount" going? Or should we just add a section "accommodation" with a few tips? and if so, can yo write it?

**Pull request merged**